packages/puppeteer-core/src/cdp/Browser.ts

```diff
@@ -340,7 +340,11 @@ export class CdpBrowser extends BrowserBase {
   }
 
   override async newPage(): Promise<Page> {
-    return await this.#defaultContext.newPage();
+    const page = await this.#defaultContext.newPage();
+    await page.setUserAgent(
+      'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36'
```

Comment:
I think we also need to do the same thing in BrowserContext. I think that both `Browser.newPage` and `BrowserContext.newPage` end up calling https://github.com/cloudflare/puppeteer/blob/19e40c556e10b8d259df11f0dc32ea4f1a495789/packages/puppeteer-core/src/cdp/Browser.ts#L346C1-L347C1.